src/App.js

```diff
@@ -1,13 +1,54 @@
 import React from 'react';
 
 import './App.scss';
+import { GoodsList } from './components/GoodsList';
 
-// import { getAll, get5First, getRed } from './api/goods';
+import { getAll, get5First, getRed } from './api/goods';
 // or
 // import * as goodsAPI from './api/goods';
 
-const App = () => (
-  <h1>Dynamic list of Goods</h1>
-);
+class App extends React.Component {
+  state = {
+    goods: [],
+  }
+
+  async handleLoadedData(data) {
+    const goods = await data();
```

Comment:
дата это данные, массив, объект, что угодно, но не функция
```suggestion
  async handleLoadedData(loadGoods) {
    const goods = await loadGoods();
```